graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -84,6 +86,7 @@ public AmqpConsumer(String hostname, int port, String virtualHost, String userna
         this.sourceInput = sourceInput;
         this.parallelQueues = parallelQueues;
         this.tls = tls;
+        this.requeueInvalid = !discardInvalid;
```

Comment:
This *might* lead to unfortunate errors in the future. Why not call the option `requeueInvalid` as well update the description of the configuration setting accordingly?